flake.nix

```diff
@@ -127,12 +127,15 @@
       overlays = {
         default = lib.composeManyExtensions [ self.overlays.${packageName} ];
         forVscode = (
-          final: _: {
-            forVscode = customeLib.${final.system}.forVscode;
-            forVscodeVersion = customeLib.${final.system}.forVscodeVersion;
-            forVscodePrerelease = customeLib.${final.system}.forVscodePrerelease;
-            forVscodeVersionPrerelease = customeLib.${final.system}.forVscodeVersionPrerelease;
-          }
+          final: _:
+          let
+            lib = {
+              forVscode = customeLib.${final.system}.forVscode;
+              forVscodeVersion = customeLib.${final.system}.forVscodeVersion;
+              forVscodePrerelease = customeLib.${final.system}.forVscodePrerelease;
+              forVscodeVersionPrerelease = customeLib.${final.system}.forVscodeVersionPrerelease;
+            };
+          in lib // { ${packageName} = lib; }
```

Comment:
Since overlays are still under preview, backward compatibility is not a concern. So you can just use:

```nix
int { ${packageName} = lib;}
```

Reply:
Alright then, I'll remove the old attrset